terraform/modules/marklogic/marklogic_cf_template.yml

```diff
@@ -710,28 +631,40 @@ Resources:
   InstanceSecurityGroup:
     Type: 'AWS::EC2::SecurityGroup'
     Properties:
-      GroupDescription: Enable SSH access and HTTP access on the inbound port
+      GroupDescription: Enable access on the inbound ports
       VpcId: !Ref VPC
       SecurityGroupIngress:
         - IpProtocol: tcp
-          FromPort: '22'
-          ToPort: '22'
+          FromPort: '80'
+          ToPort: '80'
           CidrIp: 0.0.0.0/0
         - IpProtocol: tcp
-          FromPort: '7998'
-          ToPort: '7998'
+          FromPort: '443'
+          ToPort: '443'
           CidrIp: 0.0.0.0/0
         - IpProtocol: tcp
-          FromPort: '8000'
+          FromPort: '7997'
           ToPort: '8010'
           CidrIp: 0.0.0.0/0
         - IpProtocol: tcp
-          FromPort: '7997'
-          ToPort: '7997'
+          FromPort: '8029'
+          ToPort: '8032'
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: '8040'
+          ToPort: '8044'
           CidrIp: 0.0.0.0/0
         - IpProtocol: tcp
-          FromPort: '7999'
-          ToPort: '7999'
+          FromPort: '8050'
+          ToPort: '8058'
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: '8140'
+          ToPort: '8142'
+          CidrIp: 0.0.0.0/0
+        - IpProtocol: tcp
+          FromPort: '8150'
+          ToPort: '8150'
```

Comment:
If we're making lots of changes possibly we should pull this out into terraform and just pass the id in, like you've done for the AMIs in the other stack.

Reply:
Yeah I was considering that but it won't just be the security group. This template isn't actually complete - we need listeners and target groups per port. The template only came with 9 set up for ports 8000-8008. 
It might be easier to do it with terraform's count. Though I'll probably trim this list of ports down to what we definitely need rather than keeping what they have in datamart.